dev/merge_arrow_pr.py

```diff
@@ -492,37 +491,30 @@ def load_configuration():
 
 
 def get_credentials(cmd):
-    username, password = None, None
+    token = None
 
     config = load_configuration()
     if "jira" in config.sections():
-        username = config["jira"].get("username")
-        password = config["jira"].get("password")
+        token = config["jira"].get("token")
 
     # Fallback to environment variables
-    if not username:
-        username = os.environ.get("APACHE_JIRA_USERNAME")
-
-    if not password:
-        password = os.environ.get("APACHE_JIRA_PASSWORD")
+    if not token:
+        token = os.environ.get("APACHE_JIRA_TOKEN")
 
     # Fallback to user tty prompt
-    if not username:
-        username = cmd.prompt("Env APACHE_JIRA_USERNAME not set, "
-                              "please enter your JIRA username:")
-
-    if not password:
-        password = cmd.getpass("Env APACHE_JIRA_PASSWORD not set, "
-                               "please enter your JIRA password:")
+    if not token:
+        token = cmd.prompt("Env APACHE_JIRA_TOKEN not set, "
+                              "please enter your JIRA PAT:")
 
-    return (username, password)
+    return token
 
 
 def connect_jira(cmd):
     try:
         return jira.client.JIRA(options={'server': JIRA_API_BASE},
-                                basic_auth=get_credentials(cmd))
+                                token_auth=get_credentials(cmd))
     except jira.exceptions.JIRAError as e:
+        #  token auth should not cause a captcha
```

Comment:
Could you remove the `except`?

Reply:
Done, we could inline the jira connection in line 539 but I think it is fine as is?